source/docs/software/advanced-controls/introduction/introduction-to-pid.rst

```diff
@@ -3,6 +3,11 @@
 Introduction to PID
 ===================
 
+This page provides a math-focused approach to explaining a PID controller. In parallel, in :ref:`a video from WPI is available <docs/software/advanced-controls/introduction/pid-video:PID Introduction Video by WPI>` if you prefer videos over reading.
+
+What is a PID Controller?
+-------------------------
+
 The PID controller is a commonly used feedback controller consisting of proportional, integral, and derivative terms, hence the name. This article will build up the definition of a PID controller term by term while trying to provide some intuition for how each of them behaves.
```

Comment:
"feedback controller" should be a hyperlink to the earlier section explaining the motivation behind feedback control